dosma/core/med_volume.py

```diff
@@ -646,12 +695,19 @@ def headers(self, flatten=False):
         Args:
             flatten (bool, optional): If ``True``, flattens header array
                 before returning.
+            json_dict (bool, optional): If ``True``, converts flattened headers to DICOM+JSON
```

Comment:
rename: `as_json_dict`

Reply:
`PyDicom` uses `to_json_dict`. Do you want to keep it in line with that, or change it to `as_json_dict`?

Reply:
`to_json_dict` indicates it's an action/function. `as_json_dict` is a descriptor, which is more in line of parameter names. so i would keep `as_json_dict`